e2e_test_support_files/extract_tag_numbers.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Define a function to extract and output a dependency value
+get_repository_tag() {
+  local file="$1"
+  local repo_name="$2"
+  local app="$3"
+  local value
+  value=$(grep "\"$app\":" "$file" | awk -F'"' '{print $4}')
```

Comment:
You could actually do this just with awk (slightly awkward because we need to escape the `/` in the app name):

```
value=$(awk -F'"' "/${app/\//\/}/ {print \$4}" "$file")
```